apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -361,7 +365,19 @@ export default function SectionsSetUpContainer({
           </>
         )}
       </div>
-
+      {showChildAccountPolicy && (
+        <Provider store={getStore()}>
```

Comment:
I am not sure if the redux is correctly placed here. The Notification component required redux so I just placed a provider around the notification.